### PR TITLE
CDAP-489: FileTailer Integration Tests should have better validations

### PR DESCRIPTION
**Pull request by ghost**
Improved FileTailer Integration Tests validation and moved common part to separate module.

### CODE REVIEW

**Comment by anwar6953** [comment]
Related comment:
https://github.com/caskdata/cdap-ingest/pull/90#issuecomment-61305729

**Comment by artemmakogon** [comment]
We are going to use utility StreamReader for File DropZone IT too. I reckon, that we need to use that utility to avoid duplication of code.

**Comment by anwar6953** [comment]
Also use the utility class in FileDropZoneIT and StreamClientIt as well, right?

**Comment by artemmakogon** [comment]
@anwar6953 Right

File: cdap-integration-tests/cdap-common-integration-tests/src/test/java/co/cask/cdap/utils/StreamReader.java [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    private String body;
+    long timestamp;
+
+    public String getBody() {
+      return body;
+    }
+  }
+
+  /**
+   * Builder for EventUtil
```
Comment (anwar6953) [inline cdap-integration-tests/cdap-common-integration-tests/src/test/java/co/cask/cdap/utils/StreamReader.java]: This comment is outdated, perhaps.
What is EventUtil? (renamed to streamreader)?

Comment (ghost) [inline cdap-integration-tests/cdap-common-integration-tests/src/test/java/co/cask/cdap/utils/StreamReader.java]: yes

**Comment by sreevatsanraman** [comment]
Can we add the configs to test the following cases:
a) No ssl, Security disabled
b) SSL, Security enabled
c) No Ssl, Security enabled

**Comment by ghost** [comment]
Added new test case: "b) SSL, Security enabled"
The other two test cases have been present before.

**Comment by sreevatsanraman** [comment]
Can you document how to run tests with and without auth?

**Comment by ghost** [comment]
https://github.com/caskdata/cdap-ingest/pull/94#issuecomment-61521774 This described in README.md

**Comment by ghost** [comment]
Or you want to transfer this information to BUILD.md?

**Comment by sreevatsanraman** [comment]
@ytalashkocv README.md is fine.

**Comment by sreevatsanraman** [comment]
lgtm

**Pull request merged**